src/components/AddItemsForm.js

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react';
+import { items } from '../lib/firebase';
+import '../css/components/AddItemsForm.css';
+
+const AddItemsForm = () => {
+  let [inputValue, setInputValue] = useState('');
+
+  const addToDatabase = (e) => {
+    e.preventDefault();
+    items()
+      .add({
+        name: inputValue,
+        added_on: Date.now(),
+      })
+      .then((res) => {
+        setInputValue('');
```

Comment:
Yeah, this is a nice detail to clean the input filed after a successful add to the database :heart: